pptx/opc/package.py

```diff
@@ -26,6 +29,7 @@ class OpcPackage(object):
 
     def __init__(self):
         super(OpcPackage, self).__init__()
+        self.partnames = defaultdict(int)
```

Comment:
Avoid adding instance variables to the interface of the class (in other words, make instance variables private. If you need the value outside the class, make a property for it). So this would be `self._partnames`.